<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserdataExportDialog</class>
 <widget class="QDialog" name="UserdataExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Export Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxUserpointExportSelected">
     <property name="text">
      <string>Export &amp;selected entries only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUserpointExportAppend">
     <property name="text">
      <string>&amp;Append to an already present file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
